### PR TITLE
Fixed project stats

### LINKED ISSUE #60: Fix project stats
In the project stats part, following fixes are required:

- [ ] Watching should display the length of array or should display 0 if the returned field is null.

- [ ] Delete `Mentors` and `Issues` part.

- [ ] Checkpoints should display `projectIssuessByprojectId.length`

### PR DESCRIPTION
**Pull request by prijeshb**
> Fixed #60 
> 
> * [x]  Watching should display the length of array or should display 0 if the returned field is null.
> * [x]  Delete `Mentors` and `Issues` part.
> * [x]  Checkpoints should display `projectIssuessByprojectId.length`

### CODE REVIEW

File: src/app/profile/user/details/details.component.ts [hunk 1]
```diff
@@ -68,6 +68,7 @@ export class DetailsComponent implements OnInit {
         this.isPageLoaded = true;
         this.user = data.user[0];
         this.listOfTech = data.user[0].technologies || [];
+        console.log(data.user);
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.ts]: Remove this and all other console.logs below

File: src/app/shared/idea-card/idea-card.component.html [hunk 2]
```diff
@@ -9,7 +9,7 @@ <h4 class="heading--subtitle">
     <hr class="seperator" />
 
     <div class="description-box">
-      <p class="paragraph paragraph--medium">{{ data.idea.description }}</p>
+      <p class="paragraph paragraph--medium">{{ data.idea.description || 0 }}</p>
```
Comment (WickedBrat) [inline src/app/shared/idea-card/idea-card.component.html]: Why is this required?

Comment (prijeshb) [reply src/app/shared/idea-card/idea-card.component.html]: It's just by mistake

**Comment by WickedBrat** [comment]
LGTM!

**Pull request merged**